### ISSUE TITLE
Code suggested.

### ISSUE THREAD

**Issue by Leoalv72** [VitalySvyatyuk/pocket_option_trading_bot#5]
Hi, I want to suggest a tips code for Martingale_coefficient:
The Martingale at 2.0 is not so good, after 5 step we have loss as show my tables, the Martingale coefficient to don't loss is >2.1, the best is 2.15 to obtain a 9 step, because the max bit is $1000.. with 2.16 you have 8 step. The 1rt table is the correct calculation with the real profit counted the last loss. The 2dn tables is the proposed in the original code [1, 3, 7, 15, 31, 62, 124, 249, 499, 999]
![Martingale_coefficient 2 15](https://github.com/VitalySvyatyuk/pocket_option_trading_bot/assets/172874464/86e1469d-1bc4-4b58-9957-3a1ff9bd58c8)

To do that, you need to change some code:
-MARTINGALE_COEFFICIENT = 2.15 #set to 2.15 to 9 step or high to less steps
-NUMBERS = {
    '0': '11',
    '1': '7',
    '2': '8',
    '3': '9',
    '4': '4',
    '5': '5',
    '6': '6',
    '7': '1',
    '8': '2',
    '9': '3',
    '.': '10', #add the '.' decimal to the virtual keyboard numbers
-def get_amounts():  # redefine get_amounts() code to calculate the new Martingale_coefficient
    amounts = []
    bit = 1 #Stating in $1
    amounts.insert(1, bit)
    for i in range(1, 10):
        amount = round(bit * MARTINGALE_COEFFICIENT, 2)
        amounts.insert(i, amount)
        bit = amount
    return amounts
-  amount_value = float(amount.get_attribute('value')[1:]) # get Float var
- #AMOUNTS = get_amounts(float(deposit.text))  # refresh amounts
..I dont need these line in the get_indicators() in the Win If.

I hop it's useful. Thanks to https://github.com/VitalySvyatyuk

**Comment by VitalySvyatyuk** [comment]
Hi @Leoalv72, thanks for your interest in the project. I appreciate your thoughts and beautiful table.
What I found is that the Martingale array is not very important. Either there are 8, 9, 10 elements in the list, it's not important. What is more important is the first numbers:
1 - the initial value is always 1, and there is 50% to win
3 - 25% to win this number
7 - 12.5% to win
15 - 6.25% to win
31 - 3.125% to win
These 5 cases have more entries than other 62, 124, 249 etc. Everything after 31 can be removed, we don't care too much about it.
Now, the first 5 numbers win very frequently, so their value should be boosted, they are making a main profit.

**Comment by Leoalv72** [comment]
Ok, I understand that with only 3 or 4 steps we obtain the best profit... interesting!! Thanks for the tip, I will try to use it with a little higher coefficient.
Thanks so much.

**Comment by Leoalv72** [comment]
A question... We have way to obtain the SMA values or points values for Parabolic SAR of the graph?

**Comment by VitalySvyatyuk** [comment]
@Leoalv72 yes, po_bot_indicators exactly for that, you can try there different strategies or even a mix of few strategies. Basic Python knowledge is enough.

**Comment by VitalySvyatyuk** [comment]
You can set a breakpoint during code execution to see values, or psar results. Or run a script in debug mode in Pycharm.

**Comment by VitalySvyatyuk** [comment]
Anyway if you stuck with a code, just ask me to implement your strategy.

**Comment by Leoalv72** [comment]
> Anyway if you stuck with a code, just ask me to implement your strategy.

Yes, thanks so much.

**Comment by KrisSzyp** [comment]
@Leoalv72 hello, maybe you can check at your place if this file with the Psar indicator works for you, because I wrote in the topic above with vitaly, but it is not there now, and for me it starts chromedrive, but it does not bet on anything and I do not know what the problem is 🤖🤔
